outlines/serve/openai_server.py

```diff
@@ -0,0 +1,785 @@
+import argparse
```

Comment:
Do we need a reference to vLLMs license since this is derived from

https://github.com/vllm-project/vllm/blob/b72af8f1eded6f5838be29eb6093ab0e0e0c240c/vllm/entrypoints/openai/api_server.py#L10